wsd/Auth.cpp

```diff
@@ -191,14 +194,15 @@ const std::string JWTAuth::createPayload()
 {
     std::chrono::system_clock::time_point now = std::chrono::system_clock::now();
     std::time_t curtime = std::chrono::system_clock::to_time_t(now);
-    const std::string exptime = std::to_string(curtime + 1800);
+    int expirySeconds = LOOLWSD::getConfigValue<int>("security.jwt_expiry_secs", 1800);
```

Comment:
I wonder if this timeout gets extended when the admin actively interacts and moves between the different views (I guess not), because it would encourage them to extend the default timeout to avoid re-logging in every 30 minutes, regardless of how active they had been (negating the purpose of a timeout).

Reply:
Interesting; the websocket should be persistent and give interactive updates over time - so a session doesn't expire; but I forget if switching views in the UI re-connects, or gets a new websocket or something: hopefully not. Anyhow - it's no regression for now.